shuup_tests/functional/test_discount_taxing.py

```diff
@@ -205,7 +209,7 @@ def test_3prods_services_2discounts(taxes):
             'Tax-C   0.50   45.333333333   22.666666667   68.000000000',
             'Tax-A   0.25   16.320000000    4.080000000   20.400000000',
             'Tax-B   0.20   88.541666667   17.708333333  106.250000000',
-            'Total   0.30  150.190000000   44.455000000  194.650000000']
+            'Total   0.30  150.180000000   44.455000000  194.650000000']
```

Comment:
if this is supposed to represent the sum of the previous three lines, shouldn't this be 150.19 as before (45.33 + 16.32 + 88.54)? wouldn't this show up as order lines not summing to the order total?